run2pp/DST_STREAMING_EVENT_run2pp_ana439_2024p007.yaml

```diff
@@ -337,6 +337,122 @@ DST_TRKR_TRACKS_run2pp:
 
 
 
+#==========================================================================================================================================================================
```

Comment:
Adds the single stream workflow.

Note:  b/c we have not kept this development in sync with the main branch, we need to review the workflow and make sure that we are correctly copying files over, respecting the default output directory structure, and ensure that the argument list is correctly specified.